Documentation: res.log() is only mentioned in quietReqLogger?

Hello,

I was reading the API docs, and noticed that `res.log()` is only mentioned in `quietReqLogger`:

> **quietReqLogger**: when true, the child logger available on req.log will no longer contain the full bindings and will now only have the request id bound at reqId (note: the autoLogging messages and the logger available on **`res.log`** will remain the same except they will also have the additional reqId property). default: false

Should we document `res.log()` elsewhere -- otherwise, it may give the impression of being a typo.

Thanks for the great work!

@mcollina: I have a PR ready, however, it appears there's an error on #master with the typescript types?

```
index.test-d.ts:140:3 - error TS2739: Type '{ id: string | undefined; method: string; url: string; headers: { header: string; }; remoteAddress: string; remotePort: number; raw: IncomingMessage; }' is missing the following properties from type 'SerializedRequest': params, query

140   req: {
      ~~~

  index.d.ts:68:5
    68     req: SerializedRequest;
           ~~~
    The expected type comes from property 'req' which is declared here on type 'StdSerializedResults'


Found 1 error in index.test-d.ts:140

```

--- a/README.md
+++ b/README.md
@@ -218,6 +218,7 @@ const logger = require('pino-http')({
 function handle (req, res) {
   logger(req, res)
   req.log.info('something else')
+  res.log.info('just in case you need access to logging when only the response is in scope')
   res.end('hello world')
 }
 
@@ -300,6 +301,7 @@ http.createServer((req, res) => {
   res[logger.startTime] = Date.now()
   someImportantThingThatHasToBeFirst(req, res)
   logger(req, res)
+  res.log.info('log is available on both req and res');
   res.end('hello world')
 }).listen(3000)
 ```

--- a/index.d.ts
+++ b/index.d.ts
@@ -89,5 +89,7 @@ declare module "http" {
 
     interface OutgoingMessage {
       [startTime]: number;
+      log: pino.Logger;
+      allLogs: pino.Logger[];
     }
   }

--- a/index.test-d.ts
+++ b/index.test-d.ts
@@ -138,6 +138,8 @@ const stdSerializedResults: StdSerializedResults = {
     123: {},
   },
   req: {
+    params: {},
+    query: {},
     id: canBeUndefined('id'),
     method: 'GET',
     url: 'http://0.0.0.0',
@@ -159,5 +161,8 @@ const httpServerListener: RequestListener = (request, response) => {
   request.allLogs[0].info("Request Received");
   // res[startTime] should be available
   response[startTime] = Date.now();
+  // res.log and res.allLogs should be available
+  response.log.info("Logging works on response");
+  request.allLogs[0].info("allLogs available on response");
   response.end("Hello world");
 };

